src/Http/Controllers/UserSettingsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Http\Requests\Account\User2FaCodeRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Blumilk\Meetup\Core\Services\User2FaCodesService;
+use Illuminate\Auth\AuthenticationException;
+use Illuminate\Auth\AuthManager;
+use Illuminate\View\View;
+
+class UserSettingsController extends Controller
+{
+    public function __construct(
+        protected AuthManager $authManager,
+    ) {}
+
+    public function index(): View
+    {
+        if ($this->authManager->user()->is_2fa_enable) {
+            $twoFactor = true;
+        } else {
+            $twoFactor = false;
+        }
+        return view("user.settings")
+            ->with("twoFactor", $twoFactor);
+    }
+
+    public function enable2Fa()
```

Comment:
```suggestion
    public function enable2Fa(): RedirectResponse
```
Don't forget about return types.